### PR TITLE
Manually add v0.9.0~v0.9.4 of heapbox

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10481
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/FilipBartos/heapbox
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: [MIT](https://github.com/FilipBartos/heapbox/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 10
   - Count of stars: 164
   - Count of forks: 29

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 please help me review this PR, thanks.

File: ajax/libs/heapbox/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "heapbox",
+  "version": "0.9.4",
+  "filename": "jquery.heapbox-0.9.4.min.js",
```
Comment (extend1994) [inline ajax/libs/heapbox/package.json]: You can see https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-common-conventions
> Filenames should not include a version number and be lowercase.

Or auto-update can't work well.
Please discuss with the author about the change of naming scheme, or we might have to manually add the file afterwards.

Comment (sufuf3) [inline ajax/libs/heapbox/package.json]: I'll wait for the author response to several days.(Maybe 5 to 7 days)
If the author didn't any answer, I'll add them manually, and remove the auto-update part from `package.json`.
Thanks.

**Comment by sufuf3** [comment]
@extend1994 About change of naming scheme, there is no response from the author. 
I added them manually, and remove the auto-update part from `package.json`.

@cdnjs/intern3 Please help me review this PR, thanks.

**Comment by sufuf3** [comment]
@hare1039 @clairetsai818  Please help me review this PR, thanks.

**Comment by sufuf3** [comment]
@kennynaoh 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t Please help me review this PR.
I can't request review from kennynaoh...
Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I minify the css files.
Please help me review this PR, Thank you.

**Pull request merged**